astropy/coordinates/astrom_manager.py

```diff
@@ -0,0 +1,142 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This module contains a helper function to fill erfa.astrom struct and a
+ScienceState, which allows to speed up coordinate transformations at the
+expense of accuracy.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from collections import OrderedDict
+
+import numpy as np
+
+from ..time import Time
+from .sky_coordinate import SkyCoord
+from ..utils.decorators import classproperty
+from ..utils.state import ScienceState
+# from ..utils import indent
+from .. import units as u
+from .. import _erfa as erfa
+from ..extern import six
+from .builtin_frames.utils import (
+    get_jd12, get_cip, prepare_earth_position_vel, get_polar_motion, get_dut1utc
+    )
+
+__all__ = ["transform_precision"]
+
+
+DEFAULT_PRECISION = None  # no MJD binning
+
+
+class transform_precision(ScienceState):
+    """
+    TBD
+    """
+
+    _value = 0.  # value steers the transform precision (MJD_BINNING)
+
+
+def get_astrom(frame, tcode, precision=None):
+    """
+    TBD
+    """
+
+    assert tcode in ['apio13', 'apci', 'apcs', 'apci13', 'apco13']
+
+    if precision is None:
```

Comment:
Is there a reason here not to use `units`?

Reply:
You're right, units should be used.